microcosm_pubsub/dispatcher.py

```diff
@@ -78,4 +91,6 @@ def configure_sqs_message_dispatcher(graph):
     return SQSMessageDispatcher(
         sqs_consumer=graph.sqs_consumer,
         sqs_message_handlers=sqs_message_handlers,
+        log_with_context=graph.config.sqs_message_dispatcher.log_with_context,
```

Comment:
Seems like if you set log with context to False, you are still required to wire in `graph.message_context`?